region/cleanup.json

```diff
@@ -432,8 +443,12 @@
           "sparking": {
             "runways": [
               {
-                "length": 8,
+                "length": 6,
```

Comment:
I thought Homing Geemer Room was the section of West Ocean we split off from the rest of the room. That has a basic 12-tile runway with a door on each side. What place are you at?

Reply:
The room left of that was my belief.